server/src/config/index.js

```diff
@@ -0,0 +1,20 @@
+require('dotenv').config();
+
+module.exports = {
+  port: normalizePort(process.env.PORT, '3000'),
+  env: process.env.NODE_ENV ? process.env.NODE_ENV : 'development',
```

Comment:
```suggestion
  env: process.env.NODE_ENV || 'production',
```

삼항을 사용하는 것보다 `||`를 사용하는 것이 더 간단해 보입니다.
그리고 별도의 설정이 없을 경우 기본으로 `production`으로 설정되는 것이 방어적으로 맞다고 봅니다.